src/os/bluestore/BlueStore.cc

```diff
@@ -6445,7 +6474,9 @@ void BlueStore::_txc_state_proc(TransContext *txc)
       }
       txc->shared_blobs_written.clear();
       if (cct->_conf->bluestore_sync_submit_transaction &&
-	  fm->supports_parallel_transactions()) {
+	  fm->supports_parallel_transactions() &&
+          (1 == kv_sync_thread_list.size())) {
```

Comment:
dose the rockdb support parallel access? if not, how much performance benefit the thread pool of kv_sync_thread will bring?